app/models/user.rb

```diff
@@ -32,6 +32,10 @@ def init
     self.username ||= email.split('@', 2)[0]
   end
 
+  def sent_contact_request(user)
```

Comment:
The name does not really match the function but fine for me if you add a `?` at the end of the method name because it returns a boolean

Reply:
done